### ISSUE TITLE
Simplify basic expressions

### ISSUE THREAD

**Issue by rfalke** [uxmal/reko#1121]
Subject: https://github.com/rfalke/decompiler-subjects/blob/master/from_holdec/stress_int_ia32/ia32_elf_base_and_packedbcd_inst__1_var__no_complex/by_reko.c

Output:
```
word32 inst_4_values_var_0()
{
	return -3710870412 + 3710870412;
}
```

It should be "return 0;"

**Comment by uxmal** [comment]
Unary operators are not being constant-folded.

**Comment by uxmal** [comment]
This was fixed in commit 9a281a10831b07a2eb1b994c52900cade955a2e8.
